far/strmix.hpp

```diff
@@ -188,36 +188,36 @@ bool IsCaseMixed(string_view Str);
 
 [[nodiscard]]
 bool SearchString(
-	string_view Haystack,
-	string_view Needle,
+	string_view const Haystack,
```

Comment:
https://stackoverflow.com/questions/38660537/why-is-const-unnecessary-in-function-declarations-in-header-files-for-parameters

Reply:
I used to do that too, but these days static analyzers often complain about useless qualifiers in such cases, which is annoying, but reasonable: they do not provide any extra information to the caller and can be easily confused with `const ref&` versions.